src/simulation/Simulation.cpp

```diff
@@ -2833,8 +2835,18 @@ int Simulation::try_move(int i, int x, int y, int nx, int ny)
 		}
 		break;
 	case PT_CNCT:
-		if (y < ny && (TYP(pmap[y+1][x]) == PT_CNCT || TYP(pmap[y+1][x]) == PT_ROCK)) //check below CNCT for another CNCT or ROCK
-			return 0;
+		{
+			float offsetX, offsetY; // Calculate offset from gravity
+			GetGravityField(x, y, 1.0f, 1.0f, offsetX, offsetY);
```

Comment:
You might want `sim->elements[PT_CNCT].Gravity` for `particleGrav` here.

Reply:
Fixed (Won't do much though since it needs to align to a coordinate anyways)